### PR TITLE
modified how front page looks, and added JS pages

### PR DESCRIPTION
**Pull request by thequbit**
@aztec8 take a look, and if you agree with where my head is at merge away!

### CODE REVIEW

**Comment by Nolski** [comment]
Could I get a quick screen cap?

File: bin/_header.html [hunk 1]
```diff
@@ -6,19 +6,46 @@
...
+        <li class="name">
+          <h1><a href="/about.html">About</a></h1>
+        </li>
+      </ul>
     </nav>
   </header>
-</div>
\ No newline at end of file
+</div>
+<!--        li-->
```
Comment (Nolski) [inline bin/_header.html:18]: No need to comment these out. If we don't need them, I'd just say delete them. This is why we have version control in the first play ;)

**Comment by thequbit** [comment]
On mobile -

**Comment by thequbit** [comment]
Reworking some things, closing this.

**Pull request closed without merging**